include/wavm/LoadedDynamicModule.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
```

Comment:
This class is used to encapsulate the properties of loaded dynamic modules. Once the module is created we don't really need to care about this, but it's really useful for debugging and sanity checking.